crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -147,22 +174,86 @@ impl ParsedQObject {
             }
         }
 
-        Ok(())
+        Ok(properties)
     }
 }
 
 #[cfg(test)]
-mod tests {
+pub mod tests {
```

Comment:
Why make the test module public?

Reply:
Because the `create_parsed_qobject` is reused elsewhere IIRC.

Reply:
Eventually i want to split out all the test helpers into a tree somewhere, so that tests can create examples of any component and have the expected output of any component. But for now we have some helpers in the test module for creating a basic version of the component and use it elsewhere.